### PR TITLE
docs(samples): add create and delete model

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

File: samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void testDeleteModel() {
+    // Delete the model that was just created
+    DeleteModel.deleteModel(BIGQUERY_DATASET_NAME, modelName);
+
```
Comment (athakor) [inline samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java:88]: nit : please remove this extra line

File: samples/snippets/src/test/java/com/example/bigquery/CreateModelIT.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            + "max_iteration=1, "
+            + "learn_rate=0.4, "
+            + "learn_rate_strategy='constant' "
+            + ") AS ( "
+            + "	SELECT 'a' AS f1, 2.0 AS label "
+            + "UNION ALL "
+            + "SELECT 'b' AS f1, 3.8 AS label "
+            + ")";
+    CreateModel.createModel(sql);
+
```
Comment (athakor) [inline samples/snippets/src/test/java/com/example/bigquery/CreateModelIT.java:85]: same here

File: samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            + "model_type='linear_reg', "
+            + "max_iteration=1, "
+            + "learn_rate=0.4, "
+            + "learn_rate_strategy='constant' "
+            + ") AS ( "
+            + "	SELECT 'a' AS f1, 2.0 AS label "
+            + "UNION ALL "
+            + "SELECT 'b' AS f1, 3.8 AS label "
+            + ")";
+
```
Comment (athakor) [inline samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java:71]: same here

File: samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    CreateModel.createModel(sql);
+
```
Comment (athakor) [inline samples/snippets/src/test/java/com/example/bigquery/DeleteModelIT.java:73]: same here and other places as well

**Comment by pmakani** [comment]
@athakor i have followed samples guide lines and their IT test cases based on that added spaces and lines

**Review by athakor — APPROVED** [review]
@pmakani thanks for the clarification

**Pull request closed without merging**